opds_import.py

```diff
@@ -1456,13 +1456,13 @@ def __init__(self, _db, collection, import_class,
                 )
             )
 
-        data_source = collection.data_source
+        data_source = self.data_source(collection)
         if not data_source:
             raise ValueError(
                 "Collection %s has no associated data source." % collection.name
             )
 
-        self.feed_url = collection.external_account_id
+        self.feed_url = self.opds_url(collection)
```

Comment:
The 'Feedbooks' protocol derives its OPDS URL from a 'language' setting rather than by an explicit 'URL' setting.